roles/ceph-config/templates/ceph.conf.j2

```diff
@@ -122,7 +122,7 @@ log file = /var/log/ceph/{{ cluster }}-rgw-{{ hostvars[host]['ansible_hostname']
 {%- if frontend_type == 'civetweb' -%}
 {{ radosgw_frontend_type }} port={{ _rgw_binding_socket }}{{ 's ssl_certificate='+radosgw_frontend_ssl_certificate if radosgw_frontend_ssl_certificate else '' }}
 {%- elif frontend_type == 'beast' -%}
-{{ radosgw_frontend_type }} {{ 'ssl_' if radosgw_frontend_ssl_certificate else '' }}endpoint={{ _rgw_binding_socket }}{{ ' ssl_certificate='+radosgw_frontend_ssl_certificate if radosgw_frontend_ssl_certificate else '' }}
+{{ radosgw_frontend_type }} {{ 'ssl_' if radosgw_frontend_ssl_certificate and not haproxy_frontend_ssl_termination else '' }}endpoint={{ _rgw_binding_socket }}{{ ' ssl_certificate='+radosgw_frontend_ssl_certificate if radosgw_frontend_ssl_certificate and not haproxy_frontend_ssl_termination else '' }}
```

Comment:
I don't think this is a good idea to mix haproxy variable into the ceph-rgw role.
Also we could have end to end TLS termination on both haproxy AND rgw which is not possible with this condition.

Reply:
Condition removed to allow TLS termination on both haproxy and rgw.